### PR TITLE
Add writing lab and blogging guides

### PR DESCRIPTION
**Pull request by awfrancisco**

### CODE REVIEW

**Comment by awfrancisco** [comment]
Added the product guide too. @jeremiak can you merge?

File: _config.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ navigation:
   url: /automated-testing-playbook
 - text: Before You Ship
   url: /before-you-ship
+- text: Blogging
+  url: /blogging-guide/
```
Comment (wslack) [inline _config.yml]: I think you should remove the last / to be consistent with other internal URLs

File: _config.yml [hunk 2]
```diff
@@ -46,6 +48,10 @@ navigation:
   url: /open-source-guide
 - text: Partnership Playbook
   url: /partnership-playbook
+- text: Product
+  url: /product-guide/
```
Comment (wslack) [inline _config.yml]: same as other comment - remove the trailing / ?

**Comment by wslack** [comment]
Besides those two notes (which may not be actual needs) LGTM

**Comment by ccostino** [comment]
Great, thank you @awfrancisco!

**Pull request merged**